docs/view.md

```diff
@@ -181,6 +182,16 @@ When `accessible` is `true`, the system will invoke this function when the user
 
 ---
 
+### `onAccessibilityEscape`
+
+When `accessible` is `true`, the system will invoke this function when the user performs the escape gesture.
+
+| Type     | Required |
```

Comment:
A `Platform` column is also needed here, right? with an `iOS` for its value, same as [accessibilitytraits](https://facebook.github.io/react-native/docs/view#accessibilitytraits)